src/main/java/frc/robot/subsystems/WristSubsystem.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.math.controller.PIDController;
+
+import edu.wpi.first.wpilibj.Servo;
+
+import com.revrobotics.spark.SparkMax;
+import com.revrobotics.spark.SparkLowLevel.MotorType;
+import com.revrobotics.RelativeEncoder;
+
+public class WristSubsystem extends SubsystemBase {
+
+  private final SparkMax armMotor = new SparkMax(1, MotorType.kBrushless);
+  private final RelativeEncoder armEncoder = armMotor.getEncoder();
+  
+  private final Servo servoWristMotor = new Servo(1);
+
+  private final PIDController armPID = new PIDController(0, 0, 0);
```

Comment:
Change the first value of this PIDController from 0 -> something small (0.1). We can refine it through testing, but we should have it as something to start